hi-ml-cpath/src/health_cpath/models/encoders.py

```diff
@@ -112,6 +109,12 @@ def __init__(self, pl_checkpoint_path: Path, tile_size: int, n_channels: int = 3
         super().__init__(tile_size=tile_size, n_channels=n_channels)
 
     def _get_encoder(self) -> Tuple[torch.nn.Module, int]:
+        try:
+            from SSL.lightning_modules.ssl_classifier_module import SSLClassifier
+            from SSL.utils import create_ssl_image_classifier
+            from SSL import encoders
+        except (ImportError, ModuleNotFoundError):
+            raise ValueError("SSL not found. This class can only be used by using hi-ml from the GitHub source")
```

Comment:
On this note - there's a line in `docs/source/histopathology.md` that states 'The tools for computational pathology cannot be as a Python package, but rather directly from the Git repository.'  which I think it would make sense to update in this PR